components/empty-state/empty-state-styles.js

```diff
@@ -40,12 +45,16 @@ export const emptyStateIllustratedStyles = css`
 		text-align: center;
 	}
 
-	.d2l-empty-state-action {
-		margin-top: 0.5rem;
+	.illustration-slot::slotted(*) {
+		display: none;
 	}
 
-	.d2l-empty-state-description {
-		margin: 0 auto 0.3rem;
+	.illustration-slot::slotted(svg:first-of-type) {
+		display: inline-block;
+	}
```

Comment:
`::slotted` can only be used to specify top level elements so the only way to control what goes in the slot is to block all children that are not svgs. This means that the `svg` itself needs to be the slot and it cannot be placed within a container like a `span` or a `div`. 

Ex:
```
// This will not render an svg
<div slot="illustration">
    <svg>
        ....
    </svg>
</div>
```